code/modules/admin/topic.dm

```diff
@@ -496,6 +496,19 @@
 		log_admin("[key_name(usr)] forced [key_name(M)] to say: [speech]")
 		message_admins(span_adminnotice("[key_name_admin(usr)] forced [key_name_admin(M)] to say: [speech]"))
 
+	else if(href_list["applyquirks"])
+		if(!check_rights(R_ADMIN))
+			return
+
+		var/mob/living/carbon/human/target = locate(href_list["applyquirks"])
+		if(!istype(target))
+			to_chat(usr, "this can only be used on instances of type /mob/living/carbon/human.", confidential = TRUE)
```

Comment:
```suggestion
			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human.", confidential = TRUE)
			return
```

The others seem to forget return too, weird